index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div id="welcome">
+            Welcome
+        </div>
+        <Nav id="menu">
+        <img src="./Images/icon-menu.png" alt="Menu Icon">
+        </Nav>
+    </header>
+<section class="bg">
+<div class="highlight">
+    Hey there. I’m Usama I’m a software developer
+</div>
```

Comment:
### Your Hero Title
![white-spacing](https://user-images.githubusercontent.com/93039359/186411956-1e01cb2b-0ba1-41dd-8347-2fff469019f3.png)

### Figma Hero Title
![hero-title](https://user-images.githubusercontent.com/93039359/186411984-73e0edb4-10ed-4439-a932-c59f11e041fd.png)
- [ ] Hero title should be in two colors as shown in the Figma template. Please fix it :blush:
- [ ] Your hero title should be in h1 tag instead of div tag to follow the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) of HTML for SEO purpose :+1:

Reply:
- [ ] @usamamaliq  as previously requested kindly change your header section, its design and element placement should match the Figma design given.